zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/multitenancy/MultiTenancyOverIdentityIT.java

```diff
@@ -726,7 +726,7 @@ void shouldDenyActivateJobWhenUnauthorized() {
 
       // then
       assertThat(result)
-          .failsWithin(Duration.ofSeconds(10))
+          .failsWithin(Duration.ofHours(10))
```

Comment:
:question: Why did you change this to 10 hours? Isn't that a bit much?

Reply:
No, good catch, that was for debugging (if you use a debugger then the fails within always ends up stopping you :smile:)